packages/driver/src/cypress.ts

```diff
@@ -374,8 +374,10 @@ class $Cypress {
     })
     .then(() => {
       return (new Promise((resolve) => {
-        if (this.$autIframe) {
+        if (this.$autIframe.prop('contentWindow')) {
```

Comment:
I started seeing some flake where $autIframe was not loaded and thus we see an exception when trying to access `contentWindow`. This handles that scenario.